webapp/channels/src/components/post_view/post_list/post_list.tsx

```diff
@@ -26,6 +27,9 @@ function markAndMeasureChannelSwitchEnd(fresh = false) {
     const {duration: dur1, requestCount: requestCount1} = measure('SidebarChannelLink#click', 'PostList#component');
     const {duration: dur2, requestCount: requestCount2} = measure('TeamLink#click', 'PostList#component');
 
+    measureAndReport('channel_switch', 'SidebarChannelLink#click', 'PostList#component', true);
```

Comment:
Can we add a comment here explaining why we're calling separately instead of once?

Reply:
Yep, will do